src/v2/Apps/Conversation/Components/InboxHeaders.tsx

```diff
@@ -82,14 +94,27 @@ export const InboxHeader: FC<BorderedFlexProps> = props => {
   )
 }
 
-export const DetailsHeader: FC<BorderedFlexProps> = props => {
+interface DetailsHeaderProps extends BorderedFlexProps {
+  showDetails: boolean
+}
+
```

Comment:
The same animation for large screen headers because the entire header is in a different container 👇